store/sqlstore/file_info_store.go

```diff
@@ -51,6 +55,9 @@ func (fs SqlFileInfoStore) createIndexesIfNotExists() {
 	fs.CreateIndexIfNotExists("idx_fileinfo_create_at", "FileInfo", "CreateAt")
 	fs.CreateIndexIfNotExists("idx_fileinfo_delete_at", "FileInfo", "DeleteAt")
 	fs.CreateIndexIfNotExists("idx_fileinfo_postid_at", "FileInfo", "PostId")
+	fs.CreateIndexIfNotExists("idx_fileinfo_extension_at", "FileInfo", "Extension")
+	fs.CreateFullTextIndexIfNotExists("idx_fileinfo_name_txt", "FileInfo", "Name")
+	fs.CreateFullTextIndexIfNotExists("idx_fileinfo_content_txt", "FileInfo", "Content")
```

Comment:
One question. That includes the content of the file? I mean is the whole content of a file?

Reply:
Not really, the Content is a new field added that (if the setting is enabled) is going to cointain the extracted text from the file. For example, if this is a pdf that have the text "hello world" in it, the `Content` field is going to contain the "hello world" text, not the binary data.

Reply:
What would be inside if the file has 8mb of size? All the extracted document?

Reply:
All the extracted plain-text (but only for supported files, and always behind a configuration flag).